capstone-frontend/src/components/ClubPage.jsx

```diff
@@ -1,10 +1,27 @@
 import React, { Component } from 'react';
 
 class ClubPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      response: {}
```

Comment:
nit: This should probably be `club`. Think about the name from the template's point of view, i.e. what data is being rendered. Glad to see you initializing state properties though!

Reply:
Will change, and thanks!